pkg/transformers/multitransformer.go

```diff
@@ -43,3 +55,27 @@ func (o *multiTransformer) Transform(m resmap.ResMap) error {
 	}
 	return nil
 }
+
+func (o *multiTransformer) transformWithCheckConflict(m resmap.ResMap) error {
```

Comment:
// Of the len(o.transformers)! possible transformer orderings, compare to a reversed order.
// A spot check to perform when the transformations are supposed to be commutative.
// Fail if there's a difference in the result.